flight/Libraries/math/misc_math.c

```diff
@@ -255,6 +255,43 @@ void cubic_deadband_setup(float w, float b, float *m, float *r)
 	*r = *m * powf(w, 3) + b * w;
 }
 
+/**
+ * Perform a linear interpolation over N points
```

Comment:
My apologies if I'm being overly pedantic, but I wonder if we could do better for the naming here. It's only because that since its mapping is constrained to [0,-1], it's not a full linear interpolation.